whatsnew-feed.xml

```diff
@@ -0,0 +1,21 @@
+---
+layout: null
```

Comment:
It's not obvious to me where you're using this file... What is its purpose? Are you proposing that we replace the existing `feed.xml` file with this?

Reply:
It is another channel that can be used. We had a community request to implement it.
You can add it to an RSS reader as well as `feed.xml` to track Whats New updates.
`feed.xml` contains any updates to files, and `whatsnew-feed.xml` contains only the updates highlighted by writers.

Reply:
Added an alternate link to the body per suggestion from @belbiy . It also works.